c7n/filters/health.py

```diff
@@ -29,12 +29,20 @@ class HealthEventFilter(Filter):
         types={'type': 'array', 'items': {'type': 'string'}},
         statuses={'type': 'array', 'items': {
             'type': 'string',
-            'enum': ['open', 'upcoming']
+            'enum': ['open', 'upcoming','closed']
         }})
 
     permissions = ('health:DescribeEvents', 'health:DescribeAffectedEntities',
                    'health:DescribeEventDetails')
 
+    def validate(self):
+        if self.data.get('types'):
```

Comment:
The intention of this is to avoid cases like the following:
![image](https://cloud.githubusercontent.com/assets/10120438/23173650/e105d6b2-f827-11e6-876b-9bc76c30cc38.png)
Which apparently the event type doesn't belong to the resource_type/service.

The format for all the event types is like: AWS_{service}_{Event Code}
For example:
AWS_EBS_VOLUME_LOST
AWS_EC2_PERSISTENT_INSTANCE_RETIREMENT_SCHEDULED
AWS_RDS_MAINTENANCE_SCHEDULED

So I though it might be a possible way to validate by checking if the resource_type/service is part of the event type.